client/omnichannel/directory/chats/contextualBar/ChatInfo.js

```diff
@@ -142,7 +143,7 @@ export function ChatInfo({ id, route }) {
 				</>}
 				{ts && metrics?.reaction?.fd && <>
 					<Label>{t('Queue_Time')}</Label>
-					<Info>{moment(ts).from(moment(metrics.reaction.fd), true)}</Info>
+					<Info>{formatDuration(moment(metrics.reaction.fd).diff(moment(ts)))}</Info>
```

Comment:
This is not working.

If you look at the image below, the chat started at 2:50 PM and the agent joined and replied 2:51 PM, there is no way the queue time be the following:

<img width="271" alt="Screen Shot 2021-03-31 at 2 51 46 PM" src="https://user-images.githubusercontent.com/59577424/113188735-ce2be200-9230-11eb-9ceb-4afc6c549fd0.png">

<img width="624" alt="Screen Shot 2021-03-31 at 2 51 43 PM" src="https://user-images.githubusercontent.com/59577424/113188753-d421c300-9230-11eb-828b-d95d20b3bf40.png">

Reply:
The rule is:

There is an agent assigned to the room? If so:

The queue time = difference between room.ts and room.servedBy.ts;

If not: 

The queue time = difference between room.ts and current time